web/src/settings_org.ts

```diff
@@ -491,10 +527,20 @@ function update_dependent_subsettings(property_name) {
     }
 }
 
-export function discard_realm_property_element_changes(elem) {
+export function discard_realm_property_element_changes(elem: HTMLElement): void {
     const $elem = $(elem);
     const property_name = settings_components.extract_property_name($elem);
-    const property_value = settings_components.get_realm_settings_property_value(property_name);
+    // TODO(evy): I added this safeparse (instead of parse) because there's
+    // a test for an invalid settings property, but I don't think we actually
+    // expect it to happen? So maybe we should remove the test.
+    const validated_property_name = realm_setting_property_schema.safeParse(property_name);
+    if (!validated_property_name.success) {
+        blueslip.error("Element refers to unknown property", {property_name});
```

Comment:
Yeah, don't think we ever pass invalid property here.

Reply:
Thanks! Removed the test